wg-api/best-practices.md

```diff
@@ -1,6 +1,6 @@
 # Electron API Design Guidelines
 
-The following are a set of guidelines for building Electron APIs. This document is maintained by the [API Working Group](https://github.com/electron/governance/tree/master/wg-api).
+The following are a set of guidelines for building Electron APIs. This document is maintained by the [API Working Group](./README.md).
```

Comment:
I could go either way on this one. I made it `./README.md` so that:

- It wouldn't trip up a relative link linter if we add one in the future ('./' might confuse it)
- You can navigate the docs via links in a markdown renderer, like the one in VS Code
  - With `./README.md` you'll actually jump to the file and can keep reading
  - With `./` you can't jump between files using that link, since it refers to a folder